README.md

```diff
@@ -49,7 +59,69 @@ The `fairlearn` package contains the following algorithms for mitigating unfairn
 
-## Install fairlearn
+
+### Fairlearn dashboard
+
+Fairlearn dashboard is a Jupyter notebook widget for assessing how a model's predictions impact different groups (e.g., different ethnicities), and also for comparing multiple models along different fairness and accuracy metrics.  
+
+#### Set-up and a single-model assessment
+
+To load the visualization dashboard and assess a single model's fairness and accuracy, launch the dashboard widget within a Jupyter notebook as follows:
+
+```python
+from fairlearn.widget import FairlearnDashboard
+
+# A_test containts your protected attributes (e.g., Age, Gender)
+# sensitive_feature_names containts your protected attributes names
+# y_true contains ground truth labels
+# y_pred contains prediction labels
+
+FairlearnDashboard(sensitive_features=A_test,
+                   sensitive_feature_names=['Gender', 'Age'],
+                   y_true=Y_test.tolist(),
+                   y_pred=[y_pred.tolist()])
+```
+
+
+After the launch, the widget walks the user through the assessment set-up, where the user is asked to select (1) the sensitive feature of interest (e.g., gender or age), and (2) the accuracy metric (e.g., model precision) along which to evaluate the overall model performance as well as any disparities across groups. These selections are then used to obtain the visualization of the model's impact on the subgroups (e.g., model precision for females and model precision for males).
+
+The following figures illustrate the setsup steps, where `gender` is selected as a sensitive feature and `accuracy` is selected as the accuracy metric.
+
+
+
+![Dashboard set-up](img/fairlearn-dashboard-config.png)
+
+
+
+After the set-up the dashboard presents the model assessment in two panels:
+
+
+|Panel|Description|
+|----|-----------|
+|Disparity in Accuracy| This panel shows: (1) the accuracy of your model with respect to your selected accuracy metric (e.g., `accuracy`) overall as well as on different subgroups based on your selected sensitive feature (e.g., `accuracy` for females, `accuracy` for males); (2) the disparity (difference) in the values  of the selected accuracy metric across different subgroups; (3) the distribution of errors in each subgroup (e.g., female, male). In the binary classification case, it will further split the errors into overprediction (predicting 1 when the true label is 0), and underprediction (predicting 0 when the true label is 1).
+|Disparity in Predictions|This panel shows a bar chart that contains the selection rate in each group. This means the fraction of data classified as 1 (in binary classification) and distribution of prediction values in regression.|
+
+
+![Fairness Insights](img/fairlearn-dashboard-results.png)
+
+#### Comparing multiple models
+
```

Comment:
I didn't have a chance to make a pass through this section, so it might not match the new flow.

Reply:
Now finished.